Make.config

```diff
@@ -254,7 +254,7 @@ MAC_PACKAGE_NAME_LOWER=$(shell echo $(MAC_PACKAGE_NAME) | tr "[:upper:]" "[:lowe
 MAC_PACKAGE_VERSION=3.99.$(PACKAGE_VERSION_REV).$(MAC_COMMIT_DISTANCE)
 MAC_PACKAGE_VERSION_MAJOR=$(word 1, $(subst ., ,$(MAC_PACKAGE_VERSION)))
 MAC_PACKAGE_VERSION_MINOR=$(word 2, $(subst ., ,$(MAC_PACKAGE_VERSION)))
-MAC_PACKAGE_VERSION_REV=$(PACKAGE_VERSION_REV)
+MAC_PACKAGE_VERSION_REV=9$(PACKAGE_VERSION_REV)
```

Comment:
@spouliot why is it `9$(PACKAGE_VERSION_REV)`?

Reply:
because XM 3.99.0.x has already shipped :)